great_expectations_cloud/agent/models.py

```diff
@@ -86,7 +87,7 @@ class DraftDatasourceConfigEvent(EventBase):
     config_id: UUID
 
 
-class UnknownEvent(EventBase):
+class UnknownEvent(AgentBaseModel):
```

Comment:
The UnknownEvent should not have extra fields nor the `organization_id` field so we inherit from `AgentBaseModel` instead of `EventBase`.

Reply:
This is reason enough for me to insist that we have 2 well named base models rather than relying implicitly on the `AgentBaseModel`'s behavior WRT to extras.

```python
class AgentBaseModelExtraForbid(BaseModel):
class AgentBaseModelExtraIgnore(BaseModel):

# or maybe just
class AgentBaseExtraForbig(BaseModel):
class AgentBaseExtraIgnore(BaseModel):
```

https://github.com/great-expectations/cloud/pull/323#issuecomment-2211189297